packages/web-components/src/components/rux-indeterminate-progress/rux-indeterminate-progress.scss

```diff
@@ -0,0 +1,89 @@
+:host {
+    --size: 3.75rem;
```

Comment:
Would it be difficult to change this and allow standard width/height properties? If not, can we expose this as a public custom prop and add some clarification in the docs that setting width/height won't work? The other custom props here should still be private

Reply:
I don't think so! I'll give it a shot